mypy/subtypes.py

```diff
@@ -622,7 +622,7 @@ def find_node_type(node: Union[Var, FuncBase], itype: Instance, subtype: Type) -
     if isinstance(node, FuncBase) or isinstance(typ, FunctionLike) and not node.is_staticmethod:
         assert isinstance(typ, FunctionLike)
         signature = bind_self(typ, subtype)
-        if node.is_property:
+        if node.is_property and not node.info.is_named_tuple:
```

Comment:
Hm, it looks like this will still do `bind_self` on the type, which I think might cause trouble, since it will modify the type of the field in an unexpected way. Make sure to test with multiple arguments.

Reply:
Looking at https://github.com/python/mypy/blob/master/mypy/checkmember.py#L495, it seems that only  `is_initialized_in_class` vars get the property/self binding treatment, so probably the right approach is to match that.